tox.ini

```diff
@@ -1,14 +1,18 @@
 [tox]
 envlist =
     py{35,36,37}-test
-    py{35,36,37}-{astropydev,numexpr}-test
-    py35-legacy-test
+    py{35,36,37}-{astropydev,legacy}-test
+    py{35,36,37}-cov
+    py37-numexpr-cov
```

Comment:
Just for my own understanding - the above line says there's an environment called 'py37-numexpr-cov', but where does it actually define what happens inside that environment? It automatically infers what to do just based on there being an environment named that? That's neat.

Reply:
It's defined under `[testenv]`. So everywhere in the testenv where a line is preceded by `numexpr: `, `cov: `, or nothing (its the default, like pytest), the `py37-numexpr-cov` environment will pull that information. So it's all about the precursors in the deps and commands sections